docs/Home.asciidoc

```diff
@@ -0,0 +1,180 @@
+the link:#sync-server[Sync Server] at the specified
+link:#sync-frequency[Sync Frequency] for a particular
+link:#dataset[DataSet].
+
+[[sync-server-loop]]
+=== Sync Server Loop
+
+The Sync Server Loop is a function that runs continuously in the
+link:#sync-server[Sync Server] with a 500ms wait between each run. +
+Each run it iterates over all link:#dataset-client[DataSet Clients] to
+see if a link:#dataset[DataSet] should be synced from the
+link:#dataset-backend[DataSet Backend].
+
+[[sync-client-loop]]
+=== Sync Client Loop
+
+The Sync Client Loop is a function that runs continuously in the
+link:#sync-client[Sync Client] with a 500ms wait between each run. +
+Each run it iterates over all link:#dataset-client[DataSet Clients] to
+see if a link:#dataset[DataSet] should be synced with the
+link:#sync-server[Sync Server].
+
+[[sync-frequency]]
+=== Sync Frequency
+
+In the link:#sync-client[Sync Client], this is the frequency to check
+for updates from the link:#sync-server[Sync Server] for a particular
+link:#dataset[DataSet]. +
+In the link:#sync-server[Sync Server], this is the frequency to check
+for updates from the link:#dataset-backend[DataSet Backend] for a
+particular link:#dataset[DataSet].
+
+*IMPORTANT*: It is possible to configure the frequency differently on
+the client and server. However, the frequency should be the same to
+avoid issues where:
+
+* the client is calling more frequently than the server is checking for
+updates from the link:#dataset-backend[DataSet Backend], causing
+unnecessary traffic from the client
+* the client is calling less frequently than the server is checking for
+updates from the link:#dataset-backend[DataSet Backend], causing the
+server to think that client is no longer active and dropping its
+link:#dataset[DataSet] from the cache
+
+[[dataset]]
+=== DataSet
+
+A DataSet is a collection of records being synchronized between 1 or
+more link:#sync-client[Sync Clients], the link:#sync-server[Sync Server]
+and the link:#dataset-backend[DataSet Backend].
+
+[[dataset-backend]]
+=== DataSet Backend
+
+The system of record for data being syncrhronized between the
+link:#sync-client[Sync Client] and the link:#sync-server[Sync Server]. +
+It could be anything that provides an API that can be integrated with
+from the link:#sync-server[Sync Server] e.g. a mysql database, a SOAP
+service. +
+The Sync Server exposes the link:Sync%20Server%20API[Sync Server API]
+for integrating with a DataSet Backend via link:#dataset_handler[DataSet
+Handlers]
+
+[[dataset-handler]]
+=== DataSet Handler
+
+A DataSet Handler is a function for intergrating the
+link:#sync-server[Sync Server] into a link:#dataset-backend[DataSet
+Backend]. +
+There are many handlers for doing CRUDL actions on a
+link:#dataset[DataSet] and managing collisions between
+link:#dataset-record[DataSet Records]. +
+The default implementation of these handlers uses fh.db (MongoDB backed
```

Comment:
Good point. I missed that. Looks like wiki page is outdated to portal documentation:
https://github.com/feedhenry/fh-mbaas-api/wiki/ (was changed 3 months ago)

I will update it to recent version.